test/e2e/framework.go

```diff
@@ -1030,6 +1030,20 @@ func (data *TestData) deletePodAndWait(timeout time.Duration, name string) error
 	}
 }
 
+func (data *TestData) waitForServiceRealized(timeout time.Duration, name string) error {
```

Comment:
I think when `get service` is ok, the flows and routes may not have be installed already. @antoninbas is it correct?

Before, @heanlan used "curl" to ensure Service is ready but it doesn't work for those busybox Services. Maybe we need to find another way to verify such Services.

Reply:
Yes although I haven't figured out why `curl` from Node doesn't work for busybox Services, I came up with another way that work for the `TestProxyHairpin` to connect to the busybox Service with retry, so ""curl" doesn't work for those busybox Services." is not a problem now. 

But there are other strange failures in `testProxyEndpointLifeCycle` and `testProxyServiceLifeCycle`. So I was trying to use the current method to bypass those failures introduced by check Services with `curl`. If this method is not sufficient, I'll go back and investigating those failures, also looking for other approaches.

Reply:
sorry about the delay
yes @lzhecheng is correct, the `Get` will succeed as soon as the Service is created and that doesn't mean that Antrea has installed the necessary flows and that the Service can be accessed. There can be quite a bit of delay and in that time window we can't say that the Service is "realized", so the name of this function is misleading.
My opinion is that the only way to test if a Service is actually "realized" is to try to access it. `curl` may not work if it's not an HTTP, unless you do something like this:
```
curl -v telnet://target ip address:desired port number
```